nalgebra-macros/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "nalgebra-macros"
+version = "0.1.0"
+authors = ["Andreas Longva <andreas.b.longva@gmail.com>"]
+edition = "2018"
+
```

Comment:
`Cargo.toml` needs some work: more metadata, remove the comments. License term needs to be resolved: Currently there's a discrepancy in that Cargo.toml for `nalgebra` says BSD-3 whereas `LICENSE` is Apache 2.0.